apiary.apib

```diff
@@ -3172,29 +3176,47 @@ According to the previous example, the parameters for creating Storage will be:
 
 + Response 200 (application/json)
 
-    + Attributes (array[StorageBackend])
+    + Attributes (array[StorageBackendList])
 
 
 ## Data Structures
 
 ### StorageBackendCreate
 
-+ backend: Snowflake (required) - can be Redshift or Snowflake
++ backend: Snowflake (required) - can be Redshift, Snowflake or Synapse
 + host: `sapi-37-demo.cmizbsfmzc6w.us-east-1.redshift.amazonaws.com` (required)
 + warehouse: `production` - required only for Snowflake
 + username: keboola (required)
 + password: 1234 (required)
-+ region: `us-east-1` (required)
++ region: `us-east-1` - required only for Snowflake and Redshift
 + owner: keboola (required) - associated AWS account owner
++ database: MyDataWarehouse - required only for Synapse
 
 ## StorageBackend
 
 + id: 123 (required)
++ host: sapi-37-demo.cmizbsfmzc6w.us-east-1.redshift.amazonaws.com (required)
 + backend: redshift (required)
++ region: us-east-1
+
+## StorageBackendList
+
++ id: 123 (required)
 + host: sapi-37-demo.cmizbsfmzc6w.us-east-1.redshift.amazonaws.com (required)
-+ username: keboola (required)
-+ password: 1234 (required)
++ backend: redshift (required)
++ region: us-east-1 (required)
 + owner: keboola (required)
++ stats
```

Comment:
tieto veci su nejake zabudnute co neboli v dokumnetacii, myslim ze v hlavnom PR som nevidel ze by si pridaval nejake taketo veci.

Reply:
Jo ta dokumentace je neaktuální, takže sem to fixnul.